src/css/components.css

```
@@ -1,4 +1,22 @@
 @layer components {
+  /* Headers */
+
+  .header {
+    @apply sticky top-0 z-50 px-4 py-6 hidden lg:flex bg-white;
+  }
+
+  .header.scrolled {
+    @apply sticky top-0 z-50 px-4 py-3 hidden lg:flex bg-white border-b border-gray-200;
```

```suggestion
    @apply py-3border-b border-gray-200;
```

those already come from `.header`, same for mobile :)